www/chatgpt-client/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=portfile:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        LumitoLuma chatgpt-client bd4b13db19b45c548bb8ef81a13feb42ff495680
```

Comment:
as usual, this should use a tag not a commit hash

Reply:
How would I do that if it does not exist?